graph2tac/loader/predict_server.py

```diff
@@ -420,8 +420,8 @@ def parse_args() -> argparse.Namespace:
                         help='checkpoint directory of the model')
 
     parser.add_argument('--arch', type=str,
-                        default='tf2',
-                        help='the model architecture tfgnn or tf2 or hmodel (current default is tf2)')
+                        default='tfgnn',
```

Comment:
```suggestion
                        default='tfgnn',
                        choices=['tfgnn', 'hmodel'],
```
While you're at it, maybe make the `--help` output a little nicer?

Reply:
What else do you have in mind for the output of help?

Reply:
Or were you just referring to that particular change?

Reply:
Well, this was just meant as a simple drive-by improvement. But here is a list of potential further improvements:
1. `--port` and `--host` are only used when `--tcp` is active. I'd specify this using something like `--tcp 0.0.0.0:3333`.
2. `--update-new-definitions` and `--update-all-definitions` are mutually exclusive. You can encode this with something like ` update_group = parser.add_mutually_exclusive_group()`.
3. `--log-level` and `--tf_log_level` should also have a `choices` entry.
4. Do we really need to differentiate `--with-meter` and `--progress-bar`?

Reply:
Also, `--model` is specified as optional, but I don't think it is.